portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -308,110 +308,128 @@ definition {
-			value1 = "WC WebContent Title");
-
-		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+	test ShouldExhibitUpdatedWebContent {
+		task ("Create WC via JSON"){
+			JSONWebcontent.addWebContent(
+				content = "WC WebContent Content",
+				groupName = "Test Site Name",
+				title = "WC WebContent Title");
+		}
 
-		ProductMenu.gotoPortlet(
-			category = "Content &amp; Data",
-			portlet = "Web Content");
+		task ("Go to Content Dashboard and Edit WC"){
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
 
-		WebContent.viewTitle(webContentTitle = "WC WebContent Title");
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
 
-		WebContentNavigator.gotoEditCP(webContentTitle = "WC WebContent Title");
+			ContentDashboard.editWebContent(webContentTitle = "WC WebContent New Title");
+		}
+		
+		task ("Check if content is correct") {
+			AssertTextPresent(
+				locator1 = "ContentDashboard#FIRST_ELEMENT_LISTED_MODIFIED_DATE",
+				value1 = "0 Seconds Ago");
+		}
 
-		WebContent.editCP(
-			webContentContentEdit = "WC WebContent New Content",
-			webContentTitle = "WC WebContent Title",
-			webContentTitleEdit = "WC WebContent New Title");
+		task ("Delete WC") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Content",
-			panel = "Applications",
-			portlet = "Content Dashboard");
+			WebContent.viewTitle(webContentTitle = "WC WebContent New Title");
 
-		AssertTextPresent(
-			locator1 = "ContentDashboard#FIRST_ELEMENT_LISTED",
-			value1 = "WC WebContent New Title");
+			WebContent.deleteCP(
+				siteURLKey = "test-site-name",
+				webContentTitle = "WC WebContent New Title");
+		}
 	}
 
 	@description = "Validade if a Web Content not associated with a Display Page Template is shown"
 	@priority = "5"
-	test ContentDashboardExhibitWebContent {
-		JSONLayoutpagetemplate.addDisplayPageTemplateEntry(
-			contentType = "Web Content Article",
-			displayPageTemplateEntryName = "Display Page Name",
-			groupName = "Test Site Name",
-			subType = "Basic Web Content");
+	test ShouldExhibitWebContentNotAssociatedWithDisplayPageTemplate {		
+		task ("Create WC via JSON"){
+			JSONWebcontent.addWebContent(
+				content = "WC Content",
+				groupName = "Test Site Name",
+				title = "WC Title");
+		}
 
-		JSONWebcontent.addWebContent(
-			content = "WC Content",
-			groupName = "Test Site Name",
-			title = "WC Title");
+		task ("Check if WC is not associated in Content Dashboard") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Content",
-			panel = "Applications",
-			portlet = "Content Dashboard");
+			AssertElementNotPresent(locator1 = "//span[contains(@title, 'This content has a display page')]");
```

Comment:
You cannot put a path within an assert. You have to create the path element for the Display Page icon in Content dashboard.

Reply:
You should validate that the Display Page associated icon is not present for the WC created. A way to do this is validate that just the WC is listed in the table and the Display Page associated icon is not present.